spot_driver/spot_driver/launch/spot_launch_helpers.py

```diff
@@ -177,6 +177,15 @@ def default_camera_sources(has_arm: bool, gripperless: bool) -> List[str]:
     return camera_sources
 
 
+def get_gripperless(ros_params: Dict[str, Any]) -> bool:
+    """Read the ros parameters to get the value of the gripperless parameter. Defaults to False if not set."""
+    gripperless = False
+    if "gripperless" in ros_params:
+        if isinstance(ros_params["gripperless"], bool):
```

Comment:
@khughes-bdai meta: shouldn't this fail if `gripperless` is defined to be of an unexpected type?

Reply:
How do you feel about put at the top of the file
```
_GRIPPERLESS: Literal["gripperless"]  = "gripperless"
```
then using the variable? Easier to change if needed and everything will be consistent (e.g. no typos in one)

Reply:
Sure, i'm okay with that, there's a few other hardcoded parameter names in this file that should also get updated alongside that

Reply:
updated